presentation/src/main/java/com/yapp/presentation/ui/admin/main/AdminMainViewModel.kt

```diff
@@ -45,10 +46,18 @@ class AdminMainViewModel @Inject constructor(
                     val sessions = entities.map { entity -> entity.mapTo() }
                     val upcomingSession =
                         sessions.firstOrNull { DateUtil.isUpcomingSession(it.date) }
```

Comment:
null인 부분을 처리해 주실려고 하셨다면 엘비스 연산자가 좀 더 적합해 보여요!
when은 가독성이 많이 떨어져 보입니다!

```suggestion
                    val upcomingSession =
                        sessions.firstOrNull { DateUtil.isUpcomingSession(it.date) } ?: AttendanceList.DEFAULT_UPCOMING_SESSION_ID
```

Reply:
null 처리와 업커밍 세션이 시작된 경우, 시작되지 않은 경우를 구분해 주기 위해서 처리한 부분인데 위쪽 코멘트 확인해 주실 수 있을까요? 요 부분 저도 조금 어떻게 해야 할지 모르겠어서 더 좋은 방향이 있다면 수정하면 좋을 것 같습니다 ㅠㅠ

Reply:
넵 위에 코멘트 확인했어요! 
3가지 경우이니 when문으로 두는게 좋을것 같네요
개인적인 의견인데 함수로 빼서 is~ 하면 좀 더 보기 쉬울것 같아요!